src/Nethermind/Nethermind.Blockchain.Test/TransactionSelectorTests.cs

```diff
@@ -51,110 +51,13 @@ public static IEnumerable ProperTransactionsSelectedTestCases
-
-                ProperTransactionsSelectedTestCase twoTransactionSelectedDueToValue =
-                    ProperTransactionsSelectedTestCase.Default;
-                twoTransactionSelectedDueToValue.Transactions.ForEach(t => t.Value = 400);
-                twoTransactionSelectedDueToValue.ExpectedSelectedTransactions.AddRange(twoTransactionSelectedDueToValue
-                    .Transactions.OrderBy(t => t.Nonce).Take(2));
-                yield return new TestCaseData(twoTransactionSelectedDueToValue).SetName(
-                    "Two transaction selected due to gas limit and value");
-
-                ProperTransactionsSelectedTestCase twoTransactionSelectedDueToMinGasPriceForMining =
-                    ProperTransactionsSelectedTestCase.Default;
-                twoTransactionSelectedDueToMinGasPriceForMining.MinGasPriceForMining = 2;
-                twoTransactionSelectedDueToMinGasPriceForMining.ExpectedSelectedTransactions.AddRange(
-                    twoTransactionSelectedDueToValue.Transactions.OrderBy(t => t.Nonce).Take(2));
-                yield return new TestCaseData(twoTransactionSelectedDueToValue).SetName(
-                    "Two transaction selected due to min gas price for mining");
-
-                ProperTransactionsSelectedTestCase twoTransactionSelectedDueToWrongNonce =
-                    ProperTransactionsSelectedTestCase.Default;
-                twoTransactionSelectedDueToWrongNonce.Transactions.First().Nonce = 4;
-                twoTransactionSelectedDueToWrongNonce.ExpectedSelectedTransactions.AddRange(
-                    twoTransactionSelectedDueToWrongNonce.Transactions.OrderBy(t => t.Nonce).Take(2));
-                yield return new TestCaseData(twoTransactionSelectedDueToWrongNonce).SetName(
-                    "Two transaction selected due to wrong nonce");
-
                 ProperTransactionsSelectedTestCase twoTransactionSelectedDueToLackOfSenderAddress =
                     ProperTransactionsSelectedTestCase.Default;
                 twoTransactionSelectedDueToLackOfSenderAddress.Transactions.First().SenderAddress = null;
                 twoTransactionSelectedDueToLackOfSenderAddress.ExpectedSelectedTransactions.AddRange(
                     twoTransactionSelectedDueToLackOfSenderAddress.Transactions.OrderBy(t => t.Nonce).Take(2));
                 yield return new TestCaseData(twoTransactionSelectedDueToLackOfSenderAddress).SetName(
                     "Two transaction selected due to lack of sender address");
-
-                ProperTransactionsSelectedTestCase missingAddressState = ProperTransactionsSelectedTestCase.Default;
-                missingAddressState.MissingAddresses.Add(TestItem.AddressA);
-                yield return new TestCaseData(missingAddressState).SetName("Missing address state");
-
-                ProperTransactionsSelectedTestCase complexCase = new ProperTransactionsSelectedTestCase()
```

Comment:
any reason for all these tests to be removed?

Reply:
just reworked them